src/test/java/pl/wrapper/parking/facade/domain/ParkingServiceImplFreeSpotsTest.java

```diff
@@ -0,0 +1,146 @@
+package pl.wrapper.parking.facade.domain;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.when;
+
+import java.time.LocalTime;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import pl.wrapper.parking.infrastructure.error.ParkingError;
+import pl.wrapper.parking.infrastructure.error.Result;
+import pl.wrapper.parking.pwrResponseHandler.PwrApiServerCaller;
+import pl.wrapper.parking.pwrResponseHandler.dto.ParkingResponse;
+
+@ExtendWith(MockitoExtension.class)
+public class ParkingServiceImplFreeSpotsTest {
```

Comment:
Same here, move unit tests to the main ParkingService test class

Reply:
> Same here, move unit tests to the main ParkingService test class

Should I move methods to ParkingServiceImplTests class? Or would it be better to create in class ParkingServiceImplTest nested class?